actors/infinispan-cluster/src/main/java/cloud/orbit/actors/cluster/JGroupsClusterPeer.java

```diff
@@ -184,15 +183,11 @@ public void receive(final Message msg)
                     ConfigurationBuilder builder = new ConfigurationBuilder();
                     builder.clustering().cacheMode(CacheMode.DIST_ASYNC);
 
-
                     cacheManager = new DefaultCacheManager(globalConfigurationBuilder.build(), builder.build());
 
-                    ConfigurationBuilder builder2 = new ConfigurationBuilder();
-                    builder2.clustering().cacheMode(CacheMode.REPL_SYNC);
-                    cacheManager.defineConfiguration("clusterTopologyCache", builder2.build());
-
                     // need to get a cache, any cache to force the initialization
-                    cacheManager.getCache("clusterTopologyCache");
+                    cacheManager.getCache("distributedDirectory");
```

Comment:
I think this was the only essential part.
You need to touch infinispan to get it to start working.

the other cache was never used, it could be but
ClusterPeer listens to jgroups view events and updates a list of active nodes.

Now... my initial goal was to put the node capabilities, 
with a list of all the actors the node supports, in infinispan. 
Never did because we kept locating the actors in the class path instead of making the actor list a configuration parameter.